.ci/run.sh

```diff
@@ -92,6 +92,12 @@ case "${CI_JOB}" in
 		echo "INFO: Running complete e2e kubernetes tests"
 		sudo -E PATH="$PATH" CRI_RUNTIME="containerd" bash -c "make kubernetes-e2e"
 		;;
+	"EXTERNAL_CRIO")
+		echo "INFO: Running tests on cri-o PR"
+		sudo -E PATH="$PATH" bash -c "make kubernetes"
+		sudo -E PATH="$PATH" bash -c "make kubernetes-e2e"
+		sudo -E PATH="$PATH" bash -c "make crio"
```

Comment:
Are we fine running those 3 back-to-back?  I remember we had to do some cleanups (CNI / network related) between one and another.  Isn't this needed anymore?

Reply:
No it's not :)
Our cleanup actually started to fail when #4038 was merged.
I had to fix it in the jenkins job, and it's actually why I decided to do this PR now :-)

Reply:
Perfect then!
Seems that I missed a lot of fun since the start of the vacation. :-)